pkg/client/gen.go

```diff
@@ -58,14 +61,16 @@ func (c *SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 	}
 
 	tmplVals := &templateValues{
-		E2EFocus:        cfg.E2EConfig.Focus,
-		E2ESkip:         cfg.E2EConfig.Skip,
-		SonobuoyConfig:  string(marshalledConfig),
-		SonobuoyImage:   cfg.Image,
-		Version:         buildinfo.Version,
-		Namespace:       cfg.Namespace,
-		EnableRBAC:      cfg.EnableRBAC,
-		ImagePullPolicy: cfg.ImagePullPolicy,
+		E2EFocus:             strings.Replace(cfg.E2EConfig.Focus, "'", "''", -1),
```

Comment:
I see this covered in #376 -- is this PR pending some updates?

Could you add a comment about why replacing one single quote with two single quotes is the thing to do here? I don't quite understand the replacement after looking at the YAML template change as well.

Reply:
If we merged #376 then I can rebase this and drop this from this PR. I'll add comments in other PR as it's more focused.

Reply:
Added comments in #376.

Reply:
Awesome. That was super helpful. Let's get the other one merged first so you can drop it from here.